### PR TITLE
Update readme and contributing

### PR DESCRIPTION
**Pull request by mingyli**

### CODE REVIEW

**Comment by mluogh** [comment]
Good guidelines.

However, I am also thinking perhaps we choose right now to standardize some things on the content side
i.e. require Spotify/Apple Music embed on every article with the song to allow readers to listen as they peruse.

**Review by RahulTirumalareddy — APPROVED** [review]
Looks good, Ming!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+layout:     post
+title:      on Lil Yachty - Minnesota
+author:     Ming Li
+date:       2017-10-08
+categories: lil yachty
+album_art:  assets/images/lil_boat.jpg
+---
+```
+
+## Guidelines
```
Comment (RahulTirumalareddy) [inline CONTRIBUTING.md:25]: Forgot to add this to my review, but going off of what Michael said, we should add a guideline to include some link to the music under review, either a hyperlink to the song on YouTube and/or a Apple Music/Spotify embed. If the review discusses the music video, embedding the video from YouTube is valid as well.

Comment (mingyli) [reply CONTRIBUTING.md:25]: Got it, I'll add a note

**Pull request merged**